src/javascript/expression.rs

```diff
@@ -139,6 +141,32 @@ impl<'module> Generator<'module> {
         })
     }
 
+    fn call<'a>(&mut self, fun: &'a TypedExpr, arguments: &'a [CallArg<TypedExpr>]) -> Output<'a> {
+        let fun = self.not_in_tail_position(|gen| gen.expression(fun))?;
+        let arguments = self.not_in_tail_position(|gen| {
+            call_arguments(
```

Comment:
Can use the existing wrap_args function

```suggestion
            wrap_args(
```

Reply:
wrap_args returns Document not Output, i.e. it doesn handle errors, that's because the arguments for a function definition are always renderable. however a function call can have any expression so rendering it has to handle the error cases where an unsupported expression is used.

We could use the error handling version for all but that difference was why I kept them separate

Reply:
Oh yes, thank you that makes sense